src/Rings/mpoly-ideals.jl

```diff
@@ -1944,14 +1944,13 @@ julia> grassmann_pluecker_ideal(2, 4)
 Ideal generated by
   x[1]*x[6] - x[2]*x[5] + x[3]*x[4]
 
-julia> R, x = polynomial_ring(residue_ring(ZZ, 7)[1], "x" => (1:2, 1:3), ordering=:degrevlex)
+julia> R, x = polynomial_ring(residue_ring(ZZ, 7)[1], "x" => (1:2, 1:3))
```

Comment:
@antonydellavecchia Similar to the above: I would like to remove the keyword here because it is part of the doc string which makes it look as if one needs to set the ordering like this. Can you confirm that it is not mathematically necessary to set the ordering?
Notice also the small change when printing the ideal below.

Reply:
it is mathematically necessary, i.e. if you want to remove this well need to force the ordering in the grassman pluecker ideal when we override the ring

Reply:
Okay, then we should really do this (override the ring). I don't think we can or should expect from the user to supply a ring with the correct ordering. I'll look into it.

Reply:
i agree, it shouldn't be too bad to fix

Reply:
polymake computes a groebner basis for the degrevlex ordering, so maybe we should fix `grassmann_pluecker_ideal(::Int, ::Int)` to return an IdealGens type now that these exist, since i don't believe they did when I wrote this?

Reply:
I assume that we should return an ideal with the Gröbner basis cached as `IdealGens` in the corresponding field. But then it sounds like this is independent of this pull request?

Reply:
sure, I can deal with this in  new pull request

Reply:
Great, thanks!